src/api/java/com/minecolonies/api/client/render/modeltype/ModModelTypes.java

```diff
@@ -0,0 +1,84 @@
+    public static final ResourceLocation FARMER_ID = new ResourceLocation(Constants.MOD_ID, "farmer");
+    public static final ResourceLocation FISHERMAN_ID = new ResourceLocation(Constants.MOD_ID, "fisherman");
+    public static final ResourceLocation UNDERTAKER_ID = new ResourceLocation(Constants.MOD_ID, "undertaker");
+    public static final ResourceLocation ARCHER_GUARD_ID = new ResourceLocation(Constants.MOD_ID, "archer");
+    public static final ResourceLocation KNIGHT_GUARD_ID = new ResourceLocation(Constants.MOD_ID, "knight");
+    public static final ResourceLocation BAKER_ID = new ResourceLocation(Constants.MOD_ID, "baker");
+    public static final ResourceLocation SHEEP_FARMER_ID = new ResourceLocation(Constants.MOD_ID, "sheepfarmer");
+    public static final ResourceLocation COW_FARMER_ID = new ResourceLocation(Constants.MOD_ID, "cowfarmer");
+    public static final ResourceLocation PIG_FARMER_ID = new ResourceLocation(Constants.MOD_ID, "pigfarmer");
+    public static final ResourceLocation CHICKEN_FARMER_ID = new ResourceLocation(Constants.MOD_ID, "chickenfarmer");
+    public static final ResourceLocation COMPOSTER_ID = new ResourceLocation(Constants.MOD_ID, "composter");
+    public static final ResourceLocation SMELTER_ID = new ResourceLocation(Constants.MOD_ID, "smelter");
+    public static final ResourceLocation COOK_ID = new ResourceLocation(Constants.MOD_ID, "cook");
+    public static final ResourceLocation STUDENT_ID = new ResourceLocation(Constants.MOD_ID, "student");
+    public static final ResourceLocation CRAFTER_ID = new ResourceLocation(Constants.MOD_ID, "crafter");
+    public static final ResourceLocation BLACKSMITH_ID = new ResourceLocation(Constants.MOD_ID, "blacksmith");
+    public static final ResourceLocation CHILD_ID = new ResourceLocation(Constants.MOD_ID, "child");
+    public static final ResourceLocation HEALER_ID = new ResourceLocation(Constants.MOD_ID, "healer");
+    public static final ResourceLocation TEACHER_ID = new ResourceLocation(Constants.MOD_ID, "teacher");
+    public static final ResourceLocation GLASSBLOWER_ID = new ResourceLocation(Constants.MOD_ID, "glassblower");
+    public static final ResourceLocation DYER_ID = new ResourceLocation(Constants.MOD_ID, "dyer");
+    public static final ResourceLocation MECHANIST_ID = new ResourceLocation(Constants.MOD_ID, "mechanist");
+    public static final ResourceLocation FLETCHER_ID = new ResourceLocation(Constants.MOD_ID, "fletcher");
+    public static final ResourceLocation CONCRETE_MIXER_ID = new ResourceLocation(Constants.MOD_ID, "concretemixer");
+    public static final ResourceLocation RABBIT_HERDER_ID = new ResourceLocation(Constants.MOD_ID, "rabbitherder");
+    public static final ResourceLocation PLANTER_ID = new ResourceLocation(Constants.MOD_ID, "planter");
+    public static final ResourceLocation BEEKEEPER_ID = new ResourceLocation(Constants.MOD_ID, "beekeeper");
+
+    public static BipedModelType settler;
```

Comment:
all upper case, constants

Reply:
I copied the format from [ModBuildings.java](https://github.com/ldtteam/minecolonies/blob/version/main/src/api/java/com/minecolonies/api/colony/buildings/ModBuildings.java).